Blog/migrations/0007_auto_20180413_1250.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.0.3 on 2018-04-13 12:50
+
+from django.db import migrations, models
+from django.utils.text import slugify
+
+
+def set_slugs_for_all_blogposts(apps, schema_editor):
+    title_list = []
+    Post = apps.get_model('Blog', 'Post')
+    for post in Post.objects.all():
+        # post.slug = post.title.replace(' ', '-')
```

Comment:
There's no need to keep uncommented code since we have the whole history in git.